packages/panzer/disc-fe/src/Panzer_L2Projection_impl.hpp

```diff
@@ -261,7 +261,8 @@ namespace panzer {
 
     {
       PANZER_FUNC_TIME_MONITOR("Exporting of mass matrix");
-      ghostedMatrix->fillComplete();
+      auto ghostedMap = factory.getGhostedMap(0);
+      ghostedMatrix->fillComplete(ghostedMap,ghostedMap);
```

Comment:
My guess is that `ghostedMap` is not one to one.  The domain Map and range Map (two arguments to fillComplete) _must_ be one to one.  My guess is that you want to use the row Map from the `ownedMatrix` as the domain and range Maps.

Reply:
@mhoemmen - this is fixed. thanks for catching!